src/createServer.js

```diff
@@ -4,0 +1,78 @@
+const http = require('http');
+const { detectCase } = require('./convertToCase/detectCase');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const allCases = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const errors = [];
+
+    try {
+      const [path, queryString] = req.url.split('?');
+      const params = new URLSearchParams(queryString);
+
+      const targetCase = params.get('toCase');
+      const originalText = path.slice(1);
+      const originalCase = detectCase(originalText);
+
+      if (!originalText) {
+        errors.push({
+          message:
+            // eslint-disable-next-line max-len
+            'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+        });
+      }
+
+      if (!targetCase) {
+        errors.push({
+          message:
+            // eslint-disable-next-line max-len
+            '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+        });
+      } else if (!allCases.includes(targetCase)) {
+        errors.push({
+          message:
+            // eslint-disable-next-line max-len
+            'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+        });
+      }
```

Comment:
It's good that you're checking if the provided case is supported. However, you're doing a case-sensitive comparison, which means that 'snake' and 'SNAKE' would be considered different. You might want to do a case-insensitive comparison to improve the user experience.